.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+node_js:
+- node
+install:
+- npm install
+script:
+- npm run build
+before_deploy:
+  - 'curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"'
+  - 'unzip awscli-bundle.zip'
+  - './awscli-bundle/install -b ~/bin/aws'
+  - 'export PATH=~/bin:$PATH'
+deploy:
+    - provider: s3
+      access_key_id: $AWS_ACCESS_KEY_ID
+      secret_access_key: $AWS_SECRET_ACCESS_KEY
+      bucket: "chbp-serverless"
+      local_dir: build
+      skip_cleanup: true
+      region: $AWS_DEFAULT_REGION
+      on:
+        branch: development
```

Comment:
Cool - should we add master too ?

Reply:
Was waiting on SSL cert to be approved in order to set up production in AWS. That's all approved now so will update this PR with instructions for master branch shortly